scripts/code_generation/jsoncan/example_generated/App/App_CanUtils.c

```diff
@@ -177,7 +182,7 @@ void App_CanUtils_JCT_Status_Pack(const JCT_Status_Signals* const in_msg, uint8_
     
     // Pack 12-bit signal JCT_UnsignedTester into payload (at bit 25 to bit 37).
     const int JCT_UnsignedTester_val = in_msg->JCT_UnsignedTester_value;
-    const uint32_t JCT_UnsignedTester_raw = CAN_ENCODE(JCT_UnsignedTester_val, CANSIG_JCT_STATUS_JCT_UNSIGNED_TESTER_SCALE, CANSIG_JCT_STATUS_JCT_UNSIGNED_TESTER_OFFSET, int);
+    const int32_t JCT_UnsignedTester_raw = CAN_SIGNED_ENCODE(JCT_UnsignedTester_val, CANSIG_JCT_STATUS_JCT_UNSIGNED_TESTER_SCALE, CANSIG_JCT_STATUS_JCT_UNSIGNED_TESTER_OFFSET, int);
     out_data[3] |= packShiftLeft(JCT_UnsignedTester_raw, 1, 0xfe);   // Packs bits #######_ of byte 3
     out_data[4] |= packShiftRight(JCT_UnsignedTester_raw, 7, 0x1f);   // Packs bits ___##### of byte 4
```

Comment:
I believe you need to rebase, these changes look like they're from one of my PRs

Reply:
Unresolving this, why did this change here?

Reply:
for this one I am not sure as I did rebase multiple times yet the same code was generating changing the code from uint to int so I am not exactly sure